### ISSUE TITLE
Obsidian hang at startup

### ISSUE THREAD

**Issue by net-frog** [dartungar/obsidian-simple-note-review#63]
I have added a note set with a tag. Everything works fine after the installation but after restart start obsidian gets stuck at loading plugins. I have also noticed that after the last note, I get an error find the next note and obsidian hangs in start.

**Comment by gemlad** [comment]
I'm also getting a hanging Obsidian start up after clearing a noteset yesterday so the query is now empty. 
(Windows desktop, Obsidian 1.5.11 , simple note review 1.2.4)

**Comment by gemlad** [comment]
Just deleted the empty note set in data.json and Obsidian loads fine now.

**Comment by net-frog** [comment]
Is there any update on this?

**Comment by dartungar** [comment]
@net-frog I'm tight on time right now; will look at it when I get a breather. As a temporary fix, you can do the same thing @gemlad  did and see if it fixes the issue.

**Comment by net-frog** [comment]
As far as I investigated there is no empty note with the tag I use with the set.

I will investigate it further.

**Comment by dartungar** [comment]
@gemlad @net-frog I've made a quick fix that should fix Obsidian's hanging up because of empty noteset queues - please try version 1.2.5 and check if it's indeed fixed.

**Comment by net-frog** [comment]
> @gemlad @net-frog I've made a quick fix that should fix Obsidian's hanging up because of empty noteset queues - please try version 1.2.5 and check if it's indeed fixed.

I have tried but unfortunately the issue is very much there.

**Comment by gemlad** [comment]
Yup - same. Just created a query that gives an empty note set and it's hanging on 'Loading plugins'

**Comment by dartungar** [comment]
@gemlad @net-frog  uh, turns out that was not all of the problem. How about trying new 1.2.6? I think I finally got it. Sorry for freezing your Obsidian.

**Comment by net-frog** [comment]
Thank you so much for your efforts. The issue is indeed resolved. However, at the end of the review I get an error message which is ok as the queue is empty, but instead if a tick is marked on the noteset instead of error message if would be better.

Thank you again for your great work.

![Screenshot from 2024-04-06 12-41-19](https://github.com/dartungar/obsidian-simple-note-review/assets/35987262/42f4ecd3-443e-4fd1-9987-3e2c557a4b84)
